main.py

```diff
@@ -17,6 +17,10 @@ def read_root():
 def read_root():
     return {"Hello": "World aaaaaaaaaaaaa!"}
 
+@appget("/b")
```

Comment:
이거 오타난 것 같은데요?

이렇게 수정되어야 할 것 같아요.
```suggestion
@app.get("/b")
```

Reply:
오 수정했습니다.